src/client/app/actions/topLevel.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+export const SEND_NOTIFICATION = 'SEND_NOTIFICATION';
+export const CLEAR_NOTIFICATIONS = 'CLEAR_NOTIFICATIONS';
```

Comment:
I think `show` and `hide` would be better names for these. 

I also think that `topLevel` is a bad name for this file (and the reducers). This file deals with only notifications, and it should be named something that makes that obvious.

Reply:
Agreed; either 'show' and 'hide' or 'add' and 'clear'; 'send' implies some network activity.

Reply:
I'm thinking about changing it to `show` and keeping `clear` because `hide` doesn't really encapsulate what the action is doing.